Prometheus/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@perses-dev/prometheus",
   "private": true,
-  "version": "0.4.1",
+  "version": "0.4.2",
```

```suggestion
"version": "0.5.0",
```

Because you are adding new functionalities, it's not a simple patch